### DISCUSSION TITLE
How to import the plugins

### DISCUSSION THREAD

**Discussion by Pizza-con-Queso** [JiHong88/SunEditor#1129] [Q&A]
Plugin import not working, it throws this error on console 

**GET http://[::1]:5173/node_modules/.vite/deps/suneditor_src_plugins.js?t=1670015742548&v=c7171617 net::ERR_ABORTED 504 (Gateway Timeout)**

i'm importing plugins like this
` let testingText= suneditor.create(document.getElementById('content1'), {`
` plugins: plugins,`
` templates: [
                {
                    name: 'Template-1',
                    html: '<p>HTML source1</p>'
                },
                {
                    name: 'Template-2',
                    html: '<p>HTML source2</p>'
                }
            ],`
`font : [Arial',tohoma',Courier New,Courier'],`
`fontSize : [8, 10, 14, 18, 24, 36],`
` colorList : [
                ['#ccc', '#dedede', 'OrangeRed', 'Orange', 'RoyalBlue', 'SaddleBrown'],
                ['SlateGray', 'BurlyWood', 'DeepPink', 'FireBrick', 'Gold', 'SeaGreen'],
            ],`
` katex: katex,`
` buttonList: [
                // default
                ['undo', 'redo'],
                [':p-More Paragraph-default.more_paragraph', 'font', 'fontSize', 'formatBlock', 'paragraphStyle', 'blockquote'],
                ['bold', 'underline', 'italic', 'strike', 'subscript', 'superscript', 'math','codeView'],
                ['fontColor', 'hiliteColor'],
                ['removeFormat'],
                ['outdent', 'indent'],
                ['align', 'horizontalRule', 'list', 'lineHeight'],
                ['-right', ':i-More Misc-default.more_vertical', 'fullScreen', 'showBlocks', 'preview', 'print', 'save', 'template'],
                ['-right', ':r-More Rich-default.more_plus', 'table', 'imageGallery'],
                ['-right', 'image', 'video', 'audio', 'link'],
            ]});`

Am i'm doing something wrong?

**Comment by JiHong88** [comment]
I don't know if this alone is accurate, but isn't this part a syntax error?
`Arial',tohoma',Courier New,Courier'`
